shared/src/iosMain/kotlin/io.newm.shared/internal/implementations/PreferencesDataStoreImpl.ios.kt

```diff
@@ -1,46 +1,40 @@
 package io.newm.shared.internal.implementations
 
+import com.liftric.kvault.KVault
 import io.newm.shared.internal.db.PreferencesDataStore
-import platform.Foundation.NSUserDefaults
 
 class PreferencesDataStoreImpl : PreferencesDataStore {
-
-    private val userDefaults = NSUserDefaults.standardUserDefaults
+    private val vault: KVault = KVault("newm_ios_preferences")
```

Comment:
Nice, different implementations per client but same interface/protocol. Love it!

Reply:
@cristhianescobar KVault doesn't sound familiar...? https://github.com/projectNEWM/newm-mobile/pull/241 😉